cdk/log_storage/__init__.py

```diff
@@ -0,0 +1,23 @@
+from aws_cdk import (
+    core,
+    aws_s3
+)
+
+class LogStorage(core.Stack):
+    def __init__(self, scope: core.Construct,
+                 stage: str, *, env: core.Environment):  
+              
+        super().__init__(scope, f'LogStorage-{stage}', env=env)
+
+
+        self.s3_log_bucket()
+
+
+        
+    def s3_log_bucket(self):
+        self.log_bucket = aws_s3.Bucket(self, 'quicksight-log-data',
```

Comment:
Can you also enforce SSL? https://docs.aws.amazon.com/cdk/api/v1/docs/aws-s3-readme.html#enforcing-ssl

Reply:
Used enforce_ssl option in S3 bucket creation to address this.